node/src/test/kotlin/net/corda/node/services/statemachine/StateMachineManagerTests.kt

```
@@ -111,17 +116,11 @@ class StateMachineManagerTests {
         fiber.actionOnSuspend = {
             throw exceptionDuringSuspend
         }
-        var uncaughtException: Throwable? = null
-        fiber.uncaughtExceptionHandler = UncaughtExceptionHandler { f, e ->
```

How come this is removed? Does the exception get swallowed now?